plugins/modules/identity/keycloak/keycloak_user.py

```diff
@@ -0,0 +1,524 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019, Adam Goossens <adam.goossens@gmail.com>
```

Comment:
You might want to add your own copyright here as well (in a new line). Same for the other file.